org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/JavaElementDelta.java

```diff
@@ -678,22 +683,6 @@ public String toDebugString(int depth) {
 		}
 		IResourceDelta resourceDelta = this.resourceDeltas[i];
 		buffer.append(resourceDelta.toString());
-		buffer.append("["); //$NON-NLS-1$
```

Comment:
Why that? It is 1) unrelated and 2) changes functionality

Reply:
Will fix it, I meant to delete other code instead.

Reply:
Why is this resolved? The change is still there?